http/swagger.yml

```diff
@@ -6929,4 +7041,4 @@ components:
         properties:
           type: object
           description: Key/Value pairs associated with this label. Keys can be removed by sending an update with an empty value.
-          example: {"color": "ffb3b3", "description": "this is a description"}
+          example: {"color": "ffb3b3", "description": "this is a description"}
```

Comment:
What's up with this change?

Reply:
I think it's my vs code's formatter, kill an extra blank space at the end.